maskrcnn_benchmark/modeling/box_coder.py

```diff
@@ -70,8 +70,8 @@ def decode(self, rel_codes, boxes):
         TO_REMOVE = 1  # TODO remove
         widths = boxes[:, 2] - boxes[:, 0] + TO_REMOVE
         heights = boxes[:, 3] - boxes[:, 1] + TO_REMOVE
-        ctr_x = boxes[:, 0] + 0.5 * widths
-        ctr_y = boxes[:, 1] + 0.5 * heights
+        ctr_x = boxes[:, 0] + torch.tensor(0.5, dtype=torch.float32) * widths
```

Comment:
This isn't good for general consumption. Are those needed? I cannot really imagine you cannot multiply by a scalar in ONNX.